qsirecon/interfaces/dsi_studio.py

```diff
@@ -339,8 +339,13 @@ def _post_run_hook(self, runtime):
         atlas_name = self.inputs.atlas_name
         atlas_labels_df = pd.read_table(self.inputs.atlas_labels_file)
 
+        atlas_labels_df["index"] = atlas_labels_df["index"].astype(int)
+        if 0 in atlas_labels_df["index"].values:
+            print(f"WARNING: Atlas {atlas_name} has a 0 index. This index will be dropped.")
```

Comment:
This is important, I think one of the popular neuroparc atlases has a 0 for one of its region ids

Reply:
That's really weird... So I guess we won't be able to use that particular atlas without modification?